AIDevGallery/Samples/SharedCode/EmbeddingGenerator.cs

```diff
@@ -44,6 +45,15 @@ public EmbeddingGenerator(string modelPath, HardwareAccelerator hardwareAccelera
         {
             _sessionOptions.AppendExecutionProvider_DML(DeviceUtils.GetBestDeviceId());
         }
+        else if (hardwareAccelerator == HardwareAccelerator.QNN)
+        {
+            Dictionary<string, string> options = new()
+            {
+                { "backend_path", "QnnHtp.dll" }
```

Comment:
we should include the high performance kv here too

Reply:
Added

Reply:
You removed QNN from the embedding models, so this if statement should be removed.

Reply:
I kept it in there for when we do add QNN - doesn't break anything